### PR TITLE
[frio]: Combine Link to Source with Platform icon in post view

### LINKED ISSUE #13624: [frio] Combine Link to Source with Platform icon in post view
The recent effort to improve the post view was an overdue and welcome change. Such changes will always be controversial as any UI change is. But whatever your stance on the change is, arguably the overall look is a lot cleaner now.

Which made me think about the `Link to source` UI.

**Currently**
`Link to source` is located in the top right of each post / comment and consists of the network icon (non-interactive) and a link icon (actual link to source).

**Expected**
This proposes to
- move the network icon to the left
- make the network icon interactive and link to source
- remove the link icon
- move the post info icon (public / not-listed / ...) to the second row

resulting in a less cluttered UI. Having elements to interact with in evry corner of the post is not great UX.

### PR DESCRIPTION
**Pull request by Raroun**
Updated wall_thread.tpl to match FR #13624

### CODE REVIEW

File: view/theme/frio/templates/wall_thread.tpl [hunk 1]
```diff
@@ -125,16 +125,11 @@ as the value of $top_child_total (this is done at the end of this file)
 			{{* contact info header*}}
 		<div class="contact-info">
 			<div class="preferences">
-				{{if $item.network_icon != ""}}
-					<span class="wall-item-network"><i class="fa fa-{{$item.network_icon}}" title="{{$item.network_name}}" aria-hidden="true"></i></span>
+				{{if $item.network_icon != "" && $item.plink}}
+					<span class="wall-item-network"><a href="{{$item.plink.href}}" class="plink u-url" aria-label="{{$item.plink.title}}" target="_blank"><i class="fa fa-{{$item.network_icon}} fakelink" title="{{$item.network_name}} - {{$item.plink.title}}" aria-hidden="true"></i></span>
```
Comment (MrPetovan) [inline view/theme/frio/templates/wall_thread.tpl]: The conditions on `$item.network_icon` and `$item.plink` were separate, please keep them separate in the new behavior:
- network icon and plink: The case you created
- network icon but no plink: Just the network icon like before
- plink but no network icon: Just the external link like before
- no network icon nor plink: Nothing, like before.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: view/theme/frio/templates/wall_thread.tpl (hunk 1 above)
Comment (Raroun) [reply view/theme/frio/templates/wall_thread.tpl]: I handled the new case with a new if query and wrote the remaining code in an else

Comment (MrPetovan) [reply view/theme/frio/templates/wall_thread.tpl]: Thanks, I now realized I missed the default network display when there's no icon. It should still be a link to the original post if a plink exists.

Comment (Raroun) [reply view/theme/frio/templates/wall_thread.tpl]: that should be already there?
The code on this is exactly the same ase before, i just wrapped it in an if for the new case.
Or am i getting it wrong? :)

Comment (MrPetovan) [reply view/theme/frio/templates/wall_thread.tpl]: Let's go back to our four cases:
1. Network Icon and PLink: A link to the original post showing the network icon.
2. No Network Icon but PLink: A link to the original post showing the network name.
3. Network Icon but no PLink: Just the network icon.
4. Neither Network Icon not PLink: Just the network name.

At the moment I believe case 2 isn't handled expectedly.

File: view/theme/frio/templates/wall_thread.tpl [hunk 2]
```diff
@@ -122,18 +122,19 @@ as the value of $top_child_total (this is done at the end of this file)
...
+   					<span class="wall-item-network"><a href="{{$item.plink.href}}" class="plink u-url" aria-label="{{$item.plink.title}}" target="_blank"><i class="fa fa-{{$item.network_icon}} fakelink" title="{{$item.network_name}} - {{$item.plink.title}}" aria-hidden="true"></i></a></span>
+				{{else}
+					{{if $item.plink}}
+       						<a href="{{$item.plink.href}}" class="plink u-url" aria-label="{{$item.plink.title}}" title="{{$item.network_name}} - {{$item.plink.title}}" target="_blank">{{$item.network_name}}</a>
+   					{{elseif $item.network_icon != ""}}
+       						<span class="wall-item-network"><i class="fa fa-{{$item.network_icon}}" title="{{$item.network_name}}" aria-hidden="true"></i></span>
+    					{{else}}
+        					<span class="wall-item-network" title="{{$item.app}}">{{$item.network_name}}</span>
+    					{{/if}}
 				{{/if}}
```
Comment (MrPetovan) [inline view/theme/frio/templates/wall_thread.tpl]: Condition simplification.
```suggestion
				{{if $item.network_icon && $item.plink}}
   					<span class="wall-item-network"><a href="{{$item.plink.href}}" class="plink u-url" aria-label="{{$item.plink.title}}" target="_blank"><i class="fa fa-{{$item.network_icon}} fakelink" title="{{$item.network_name}} - {{$item.plink.title}}" aria-hidden="true"></i></a></span>
				{{elseif $item.plink}}
       					<a href="{{$item.plink.href}}" class="plink u-url" aria-label="{{$item.plink.title}}" title="{{$item.network_name}} - {{$item.plink.title}}" target="_blank">{{$item.network_name}}</a>
   				{{elseif $item.network_icon}}
       					<span class="wall-item-network"><i class="fa fa-{{$item.network_icon}}" title="{{$item.network_name}}" aria-hidden="true"></i></span>
    				{{else}}
        				<span class="wall-item-network" title="{{$item.app}}">{{$item.network_name}}</span>
    				{{/if}}
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]
Nice, we're almost there!

**Comment by Raroun** [comment]
i noticed there are some other places too - for example in shared_content.tpl. I will look into the other places in this week.
As allways - thanks for the reviews and for your patience :)

**Review by MrPetovan — APPROVED** [review]

**Comment by MrPetovan** [comment]
And thank you for your work!

**Comment by loma-one** [comment]
I have noticed that there is another template in which source and link are separated.
I noticed it in a shared post.

**Comment by Raroun** [comment]
@loma-one 
I´l fix it this weekend.

**Comment by annando** [comment]
Also these icons currently change their colour, when the mouse is over the message:
image

**Comment by MrPetovan** [comment]
That's good, they are links and should behave like other links.

**Comment by foss-** [comment]
> Also these icons currently change their colour, when the mouse is over the message

Here are a few thoughts on that and icon color use in the post info icons: https://github.com/friendica/friendica/issues/13645

**Comment by annando** [comment]
@MrPetovan 
> That's good, they are links and should behave like other links.

Currently they don't:
image

The name is coloured red but the date string is underlined.

**Comment by foss-** [comment]
> Currently they don't: image
> 
> The name is coloured red but the date string is underlined.

On the danger of sounding like a broken record player and I know that issue has a lot of text, but it does cover the problem you found, which is unrelated to the issue here covering removing the hyperlink icon and moving the publicity icon to the second row. Issue for color inconsistencies is https://github.com/friendica/friendica/issues/13645
I'll add screenshots to better get the points across.

**Comment by Raroun** [comment]
@annando 
This happens because i set the "fakelink" attribute to the Icons.
Otherwise they would be underlined and allways hilighted, which doesn´t look good.
There are some browsers like Safari which are ignoring the fakelink attribute tho.

**Pull request merged**